include/z64.h

```diff
@@ -654,10 +654,10 @@ typedef struct {
     /* 0x021A */ s16    heartsEnvR[2];
     /* 0x021E */ s16    heartsEnvG[2];
     /* 0x0222 */ s16    heartsEnvB[2];
-    /* 0x0226 */ s16    unk_226;
-    /* 0x0228 */ s16    unk_228;
-    /* 0x022A */ s16    unk_22A;
-    /* 0x022C */ s16    unk_22C;
+    /* 0x0226 */ s16    unk_226; // Used only in unused functions
+    /* 0x0228 */ s16    unk_228; // Used only in unused functions
+    /* 0x022A */ s16    beatingHeartOscillator;
+    /* 0x022C */ s16    beatingHeartOscillatorDecrease;
     /* 0x022E */ s16    unk_22E;
```

Comment:
(sorry again for the late review on your things)
I would probably call this a direction, and same for `heartColorOscillatorDecrease`, and not treat it as a bool. I think there was some conversation about this initially that I may have missed. But this pattern is pretty common throughout interface related things and so far I think we've pretty consistently been treating it as a pulsing direction